sdk-quarkus/kar-rest-client/.mvn/wrapper/MavenWrapperDownloader.java

```diff
@@ -0,0 +1,117 @@
+/*
```

Comment:
I suspect the entire .mvn subdirectory should not be committed to git.

Reply:
Makes sense, I'll delete the generated maven artifacts.